FieldOpt/Reservoir/grid/cell.cpp

```diff
@@ -54,43 +53,42 @@ bool Cell::Equals(const Cell &other) const
 bool Cell::EnvelopsPoint(Eigen::Vector3d point) {
     bool point_inside = true;
     for (Face face : faces_) {
-        if ((point - face.corners[0]).dot(face.normal_vector) < 0)
+        double dot_prod = (point - face.corners[0]).dot(face.normal_vector);
+        if ( dot_prod < 0)
             point_inside = false;
     }
     return point_inside;
 }
 
 void Cell::initializeFaces() {
-    int face_indices_points[6][4] = {
-        {0, 2, 1, 3},
-        {4, 5, 6, 7},
-        {0, 4, 2, 6},
-        {1, 3, 5, 7},
-        {0, 1, 4, 5},
-        {2, 6, 3, 7}
-    };
+            int face_indices_points[6][4] = {
+                     {0, 2, 1, 3},
+                     {4, 5, 6, 7},
+                     {0, 4, 2, 6},
+                     {1, 3, 5, 7},
+                     {0, 1, 4, 5},
+                     {2, 6, 3, 7}
+            };
+
+
+//    int face_indices_points[6][4] = {
+//        {2, 0, 3, 1},
+//        {6, 7, 4, 5},
+//        {2, 6, 0, 4},
+//        {3, 1, 7, 5},
+//        {2, 3, 6, 7},
+//        {0, 4, 1, 5}
+//    };
 
     for (int ii = 0; ii < 6; ii++) {
         Face face;
         face.corners.push_back(corners_[face_indices_points[ii][0]]);
         face.corners.push_back(corners_[face_indices_points[ii][1]]);
         face.corners.push_back(corners_[face_indices_points[ii][2]]);
         face.corners.push_back(corners_[face_indices_points[ii][3]]);
-
-        face.normal_vector =
-            (face.corners[2] - face.corners[0]).cross(
-                face.corners[1] - face.corners[0]).normalized();
-
+        face.normal_vector = (face.corners[2] - face.corners[0]).cross(face.corners[1] - face.corners[0]).normalized();
```

Comment:
I'll do something in the middle, but newline immediately after `=` is a no-no in my book.